modules/dedicated-managing-dedicated-administrators.adoc

```diff
@@ -6,24 +6,11 @@
 =  Managing {product-title} administrators
 
 Administrator roles are managed using a `dedicated-admins` group on the cluster.
-Existing members of this group can edit membership. To view a list of current
-dedicated administrators by user name, you can use the following command:
+Existing members of this group can edit membership via the OpenShift Cluster
```

Comment:
Recommend linking to OCM:  https://cloud.redhat.com/openshift

Reply:
Agreed - added a link.